composer.json

```diff
@@ -24,19 +24,11 @@
       }
     ],
     "require": {
-        "wp-coding-standards/wpcs": "2.1.1",
+        "wp-coding-standards/wpcs": "~2.3.0",
         "squizlabs/php_codesniffer": ">=3.3.1 <3.4.0",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0"
     },
     "extra": {
         "phpcodesniffer-search-depth": 5
-    },
-    "authors": [
-        {
-            "name": "Aubrey Portwood",
-            "email": "aubrey@webdevstudios.com",
-            "homepage": "http://webdevstudios.com/",
-            "role": "Developer"
-        }
-    ]
```

Comment:
This was a duplicate. I have another authors config that has both WDS and me listed.